ogr/services/github.py

```diff
@@ -269,6 +270,24 @@ def pr_create(
         )
         return self._pr_from_github_object(created_pr)
 
+    def update_pr_info(self, pr_id: int, title: str, description: str):
+        """
+        Update pull-request information.
+
+        :param pr_id: int The ID of the pull request
+        :param title: str The title of the pull request
+        :param description str The description of the pull request
+        :return: PullRequest
+        """
+        pr = self.github_repo.get_pull(number=pr_id)
```

Comment:
What about the situation, when the PR does not exist? I would like to get some clear exception about that.

Please, check that situation and raise an ogr-specific exception (you can create a new one for GitHub like we have `PagureAPIException` in ./ogr/exceptions.py) with the message about PR not found...